docs/settings/security-settings.asciidoc

```diff
@@ -370,7 +358,31 @@ xpack.security.audit.appender:
 <1> Rotates log files every 24 hours.
 <2> Keeps maximum of 10 log files before deleting older ones.
 
-| `xpack.security.audit.appender.type`
+[NOTE]
+============
+{ess} does not support custom log file policies. To enable audit logging on {ess} only specify:
```

Comment:
This is also a limitation for ECE, not just ESS. I'm not sure if we have an `{ece}` token though? @gchaps WDYT?